src/main/resources/templates/index.html

```diff
@@ -11,7 +11,11 @@
 <div class="bg-img">
     <h1 hidden>Welcome to Treasure Task</h1>
 
-    <div class="form-signin w-100 m-auto modal-content rounded-4 shadow">
+    <div class="scroll-img m-auto">
+        <p class="hp-about">With Treasure Task, parents can assign chores to their children to monitor weekly tasks. To better encourage children to not only complete their chores but also do them well, each chore can be worth a determined amount of points. Children can collect enough points to redeem rewards set by their parents. Child accounts can be created to help older children keep track of chores and rewards themselves, giving them a sense of responsibility.</p>
+    </div>
+
+    <div class="form-signin w-100 m-auto modal-content rounded-4 shadow" id="login">
         <form method="post" th:action="@{/rewards/create}">
```

Comment:
```suggestion
        <form method="post" th:action="@{/login}">
```
Don't know if we need the `th:action` part on the form but I definitely feel like we need to change it from `/rewards/create` (not that it seems to be breaking it, but still)